config/secrets.yml

```
@@ -27,3 +27,8 @@ production:
   secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
   bjc_password: <%= ENV["BJC_PASSWORD"] %>
   piazza_password: <%= ENV["PIAZZA_PASSWORD"] %>
+
+staging:
+  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
+  bjc_password: <%= ENV["BJC_PASSWORD"] %>
+  piazza_password: <%= ENV["PIAZZA_PASSWORD"] %>
```

```suggestion
  <<: *production
```

This would reduce some duplication